tests/e2e/cypress/integration/spanish.spec.js

```diff
@@ -0,0 +1,49 @@
+describe("Index page - Internationalization spec", () => {
+  it("Gives user Spanish language option", () => {
+    cy.visit("/")
+
+    cy.get("#header").should("contain", "Español")
+    cy.get("#header").should("not.contain", "English")
+  })
+
+  it("Changes the language to Spanish", () => {
+    cy.visit("/")
+
+    cy.contains("Español")
+      .click()
+
+    cy.contains("La nueva forma de pagar")
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
```

Comment:
Let's also verify that the URL stayed the same (i.e. clicking the button didn't take them to another page)

Reply:
To be resolved in different PR.